content/docs/other-guides/usage-reporting.md

```diff
@@ -20,24 +20,28 @@ command only restarts spartakus with reportUsage set to `false`
 kubectl -n ${NAMESPACE} delete deploy -l app=spartakus
 ```
 
-**Reporting usage data is one of the most significant contributions you can make to Kubeflow; so please consider turning it on.** This data
-allows us to improve the project and helps the many companies working on Kubeflow justify continued investment.
+**Reporting usage data is one of the most significant contributions you can make to Kubeflow; so please consider turning it on.** This data allows us to improve the project and helps the many companies working on Kubeflow justify continued investment.
 
-You can improve the quality of the data by giving each Kubeflow deployment a unique id by editing `spartakus.yaml` in your `KF_APP` directory:
+**To prevent Spartakus from being deployed,** do the following before running `kfctl apply`:
```

Comment:
Add an empty line after this line, to make the list render properly. To see how your page is rendering, take a look at the preview here: https://deploy-preview-1085--competent-brattain-de2d6d.netlify.com/docs/other-guides/usage-reporting/